### ISSUE TITLE
GIT

### ISSUE THREAD

**Issue by archistico** [yoanbernabeu/Roadmap-Dev-Symfony#74]
Commandes GIT de base
- Git config
- Git init
- Git add
- Clone git
- Git commit
- Git status
- Git push
- Git checkout
- Git remote
- Git branch
- Git pull
- Git merge
- Git diff
- Git tag
- Git log
- Git reset
- Git rm
- Git stash
- Git show
- Git fetch
- Git ls-tree
- Git cat-file
- Git grep
- Git gc
- git archive
- Git prune
- Git fsck
- Git rebase

**Comment by leknoppix** [comment]
Je pense qu'il y a encore pas mal de chose à dire et à faire dans cette issue, je ne pense pas qu'il faut la clore.

**Comment by yoanbernabeu** [comment]
> Je pense qu'il y a encore pas mal de chose à dire et à faire dans cette issue, je ne pense pas qu'il faut la clore.

Oui tu as raison.
Une fois la structure de base de la Roadmap, nous pourrons re ouvrir des issues plus détaillée sur les parties a produire.

Dans un premier temps, j'essaye d'orienter les contributions sur la structure globale de la Roadmap 😅

**Comment by Grezor** [comment]
Est-ce que je peux créer une page dans la **section Git** et mettre les alias git.

**Comment by leknoppix** [comment]
> Est-ce que je peux créer une page dans la **section Git** et mettre les alias git.

Oui, ce n'est pas parce qu'un issue est close que c'est définitivement fermé.
Ajoute ce que tu veux, l'ami @yoanbernabeu validera ou pas tes ajouts.

**Comment by yoanbernabeu** [comment]
> Est-ce que je peux créer une page dans la **section Git** et mettre les alias git.

Hello, comme le dit @leknoppix toutes les contributions sont les bienvenus !

(Juste, pour ta PR soit acceptée, je vais faire un article prochainement pour l'expliquer, l'idée c'est de ne pas mettre du contenu "brut", mais de référencer du contenu pertinent déjà existant :) )

Merci en tous cas !
